### PR TITLE
fix(amplify_flutter) cites correct minSDK version for android in flutter

### PR DESCRIPTION
**Pull request by haverchuck**
*Issue #, if available:*
#2751 

*Description of changes:*
Bumps the minSDK version for amplify-flutter w/ Android.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/lib/project-setup/fragments/flutter/prereq/prereq.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
 - Install [Flutter](https://flutter.dev/docs/get-started/install) version 1.20.0 or higher
 - Setup your [IDE](https://flutter.dev/docs/get-started/editor?tab=androidstudio)
-- Android API level 16 (Jelly Bean) or higher
+- Android API level 21 (Android 5.0) or above
```
Comment (jpignata) [inline docs/lib/project-setup/fragments/flutter/prereq/prereq.md:3]: This is typically known as Lollipop, I believe.

**Review by jpignata — APPROVED** [review]
Have we made the product decision to not support <21 on Flutter? Might be worth a quick discussion as we're seeing increased hurdles on Android between 16 - 21 these days and I'd be curious what trade-offs were identified.

**Review by Ashish-Nanda — APPROVED** [review]

File: docs/lib/project-setup/fragments/flutter/prereq/prereq.md (hunk 1 above)
Comment (haverchuck) [reply docs/lib/project-setup/fragments/flutter/prereq/prereq.md:3]: @jpignata  I'm leaving it at Android 5.0 for now to align with the wording in other sections of our docs.

**Review by jamesonwilliams — APPROVED** [review]

**Review by mauerbac — APPROVED** [review]
Lgtm

**Pull request merged**